src/shims/x86/mod.rs

```diff
@@ -534,6 +540,61 @@ fn shift_simd_by_scalar<'tcx>(
     Ok(())
 }
 
+/// Shifts each element of `left` by the corresponding element of `right`.
+///
+/// For logic shifts, when right is larger than BITS - 1, zero is produced.
+/// For arithmetic right-shifts, when right is larger than BITS - 1, the sign
+/// bit is copied to remaining bits.
```

Comment:
which remaining bits? Isn't the sign bit right shifted together with the other bits, so right shifts larger than `BITS - 1` are equivalent to a right shift of `BITS - 1`?

Reply:
I meant all bits, fixed